docs/content/getting-started.md

```diff
@@ -269,13 +269,13 @@ evaluating multiple branches in a single `switch` statement. The first
 (top-to-bottom) `case` with a true `predicate` is the one that is evalauted.
 {{< /note >}}
 
-*predicate*s are evaluated like other templates in converge, and may reference
-`param`s, and perform `lookup`s on other values in the system.  A *predicate*
-may not reference any of it's *child* resourceses.  The value of the predicate
-is it's truth value:  The strings `t` and `true` (case insensitive) are `true`
-values and will cause the *branch* to be evaluated.  The strings `f` and `false`
-(case insensitive) will cause the *branch* to remain unevaluated.  Any other
-value is an error.
+A predicate is a special kind of go text template.  It's evaluated in two
+passes. In the first pass `lookups` and `params` are rendered as they would be
```

Comment:
Please add a comma, and the "s" making lookup and param plural should be outside  quotes:
"In the first pass, `lookup`s and `param`s..."